tests/test_omen.py

```diff
@@ -931,3 +931,29 @@ def test_cache_sharing():
     db.delete("cars", id=45)
     cache.reload()
     assert not cache.select_one(id=45)
+
+
+def test_cache_sharing_threaded():
```

Comment:
probably should add a test where you remove one item from the db, and assert it's not in the cache

Reply:
`test_cache_sharing` (above) does exactly that, but in a single thread. is that what you meant?

Reply:
yeah, i didn't see the case where there is no "where" clause set